source/manual/howto-backup-and-restore-in-aws-rds.html.md

```diff
@@ -45,59 +45,57 @@ Ensure that the right database ARN has been stored by:
 echo ${snapshot_arn}
 ```
 
-### Step 2
+### 2. Find which database the snapshot was generated by
 
-Find which database the snapshot was generated by getting the DBInstanceIdentifier by:
-
-Example:
+You can get this using the `DBInstanceIdentifier`, for example:
 
 * db_instance_identifier=local-links-manager-postgres
 
 ```
 gds-cli aws govuk-<environment>-admin aws rds describe-db-snapshots --db-snapshot-identifier ${snapshot_arn} --snapshot-type automated --query 'DBSnapshots[].DBInstanceIdentifier'
 ```
 
-Store the DBInstanceIdentifier as a variable
+Store the DBInstanceIdentifier as a variable:
 
 ```
 db_instance_identifier=<DB_Instance_Identifier>
 ```
 
-### Step 3
+### 3. Ensure the restored database has the same security groups
 
-The restored database must have the same security groups and be in the same VPC (that's the "subnet group name" parameter) as the original one, otherwise, apps won't be able to connect to it. Therefore the database needs to be restored in the same VPC and with the same security groups as the original instance, the snapshot came from.
+The restored database must have the same security groups and be in the same VPC (that's the "subnet group name" parameter) as the original one, otherwise, apps won't be able to connect to it. Therefore the database needs to be restored in the same VPC and with the same security groups as the original instance the snapshot came from.
 
-After running the command below, you now have all the parameters you need (snapshot-arn, db-instance-identifier, security-group-id, db-parameter-group-name, and db-subnet-group-name) to restore the database and change the restored database's security groups to match the original's.
+After running the command below, you now have all the parameters you need (snapshot-arn, db-instance-identifier, security-group-id, db-parameter-group-name, and db-subnet-group-name) to restore the database and change the restored database's security groups to match the originals.
```

Comment:
The apostrophe usage is correct - but we could change this to `to match that of the original` if there's any confusion.